bin/install_swiftpm_dependencies

```diff
@@ -1,30 +1,91 @@
 #!/bin/bash -eu
 
-CUSTOM_PACKAGE_RESOLVED_PATH=${1-}
-BUILD_TYPE=${2-}
+print_usage() {
+  echo "Usage:"
+  echo "  Using the \`Package.resolved\` of an Xcode \`.xworkspace\`:"
+  echo "    $0 --workspace PATH --scheme NAME"
+  echo "  Using the \`Package.resolved\` of an Xcode \`.xcodeproj\`:"
+  echo "    $0 --project PATH"
+  echo "  Using the \`Package.resolved\` at the root, managed by \`swift package\` (and not Xcode)"
+  echo "    $0 --use-spm"
+}
+
+# Parse input parameters
+XCWORKSPACE_PATH=
+SCHEME_NAME=
+XCODEPROJ_PATH=
+USE_SPM=false
+while [[ "$#" -gt 0 ]]; do
+	case $1 in
+		--workspace)
+			XCWORKSPACE_PATH="$2"
+      shift
+			;;
+		--scheme)
+			SCHEME_NAME=$2
+      shift
+			;;
+    --project)
+      XCODEPROJ_PATH=$2
+      shift
+      ;;
+    --use-spm)
+      USE_SPM=true
+      ;;
+    -h|--help)
+      print_usage
+      exit 0
+      ;;
+    *)
+      echo "Unknown option or unexpected argument: $1"
+      print_usage
+      exit 2
+      ;;
+	esac
+	shift
+done
```

Comment:
Looks like the indentation here uses a mix of tabs and spaces, where the rest of the diff uses spaces.

Reply:
Nice catch! Fixed in 407c0b814f44967d7dc181e98a36fa17f800e350